custom/conf/app.example.ini

```diff
@@ -1664,6 +1664,47 @@ ROUTER = console
 ;; convert \r\n to \n for Sendmail
 ;SENDMAIL_CONVERT_CRLF = true
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;[incoming_email]
```

Comment:
What about `[email.incoming]` instead?
That conforms more to the notation used for other sections.